Aztec/Classes/NSAttributedString/HTML/HTMLRepresentation.swift

```diff
@@ -0,0 +1,4 @@
+import Foundation
+
+protocol HTMLRepresentation {
+}
```

Comment:
(Still wondering if we want this guy around). If we do, maybe just move it to `HTMLElementRepresentator.swift` as well?